### DISCUSSION TITLE
Cache with Python API

### DISCUSSION THREAD

**Discussion by MatheusTramonti** [duckdb/duckdb#6997] [Q&A]
Hi Guys!
I have a doubt if, for example, when I read a csv/parquet it will save this reading in a cache or if every time it will run again without considering the previous one. If it considers the cache, how do I disable it in the API?

**Comment by Alex-Monahan** [comment]
DuckDB will re-read your parquet file each time! If you want to cache the metadata of the file, you can enable a pragma to do that: https://duckdb.org/docs/sql/pragmas#enable_object_cache-disable_object_cache

**Comment by kacar7** [comment]
import pandas as pd
pd.options.io.cache_enabled = False

This will disable the cache for all read operations in your Python session.
